src/Desktop/Utility/ExtensionMethods.cs

```diff
@@ -183,7 +183,8 @@ public static double GetDPI(this Rectangle rc)
         /// <returns></returns>
         public static double GetDPI(int left, int top)
         {
-            NativeMethods.GetDpi(new System.Drawing.Point(left, top), DpiType.Effective, out uint dpiX, out uint dpiY);
+            var win32Helper = new Win32Helper();
```

Comment:
>Win32Helper [](start = 34, length = 11)

This object will get created and destroyed many times during the process. We also create one in ExtensionMethods.cs. That seems a little bit odd for a class that will be functionally identical in each and every instance